src/Components/SearchPage/SearchPage.js

```diff
@@ -1,50 +1,61 @@
 import React, { Component } from 'react';
 import './SearchPage.css';
+import './Marquee/Marquee.css';
 import './SideBar/SideBar.css';
 import './Pokemon/PokemonList.css';
 import './Loading.css';
 import request from 'superagent';
-import MarqueeScroll from './MarqueeScroll.js';
+import MarqueeScroll from './Marquee/MarqueeScroll.js';
 import PokemonList from './Pokemon/PokemonList';
 import SideBar from './SideBar/SideBar.js';
 export default class SearchPage extends Component {
     state = {
         // Global //
         pokeData: [],
         loading: false,
-        // Search //
+        totalPokemon: 0,
+        currentPage: 1,
+        // Search/Sort/Filter //
         searchQuery: '',
-        // Sort //
+        perPage: 25,
         sortBy: ['pokemon', 'type_1', 'attack', 'defense', 'hp'],
         sortSelected: 'pokemon',
         sortOrder: '',
-        // Filter (radio) //
         pokeTypes: [],
         radioSelected: '',
     }
     componentDidMount = async () => {
         await this.fetchPokemonAndTypes()
-        // await this.fetchTypes()
     }
+    // componentDidUpdate = async (prevState) => {
+    //     const oldPageNumber = prevState.currentPage;
+    //     const newPageNumber = this.state.currentPage;
+
+    //     if (oldPageNumber !== newPageNumber) {
+    //         await this.fetchPokemonAndTypes();
+    //     }
+    // }
+
     fetchPokemonAndTypes = async () => {
-        this.setState({
+        await this.setState({
             loading: true,
             pokeData: [],
+            perPage: 25,
             pokeTypes: [],
         })
 
-        const pokeData = await request.get('https://pokedex-alchemy.herokuapp.com/api/pokedex?perPage=50')
+        const pokeData = await request.get(`https://pokedex-alchemy.herokuapp.com/api/pokedex?pokemon=${this.state.searchQuery}&page=${this.state.currentPage}&perPage=${this.state.perPage}`)
         const data = await request.get('https://pokedex-alchemy.herokuapp.com/api/pokedex/types')
 
         let pokeTypes = data.body.map(type => type.type);
 
-        this.setState({
```

Comment:
I feeeeel like you need to set state for currentpage here and that's why next/prev aren't working